content/documentation/pages/6-recipes/6-multi-platform-deployment/1-multiple-platform-accounts.md

```diff
@@ -0,0 +1,531 @@
+---
+path: 'recipes/multi-platform-deployment/multiple-platform-accounts'
+title: 'Role of Multiple Platform Deployments'
+description: 'A walk-through of multiple platform requirements and the configurations for Cloud Foundry and Kubernetes'
+---
+
+# Role of Multiple Platform Deployments
+
+The goal of this recipe is to unpack the scenarios for when multiple platform deployments becomes a necessity. Let's start
```

Comment:
I think we can remove `for` and say something like:

The goal of this recipe is to unpack the scenarios when multiple platform deployments become a necessity